downstream/modules/platform/ref-controller-inventory-sync-details.adoc

```diff
@@ -0,0 +1,27 @@
+[id="controller-inventory-sync-details"]
+
+= Inventory sync details
+
+Access the *Details* tab to view details about the job execution:
+
+image::ug-show-job-details-for-inv-sync.png[Show job details for inventory sync]
+
+You can view the following details for an executed job:
+
+Status:: It can be any of the following:
```

Comment:
Should this be bulleted?

Reply:
I added it as a definition list so it is not bulleted.